portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +30,54 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> messages = new ArrayList<>();
+  private String json;
+
+  /** Converts Messages ArrayList to Json */
+  public void toJson() throws IOException {
+    json = new Gson().toJson(messages);
+  }
+
+  /** Redirection to the main page */
+  private void doRedirect(HttpServletResponse response) throws IOException {
+    response.sendRedirect(
+      "/../../../../../../index.html"
+    );
+  }
+
+  /** Write a new message to DataStore */
+  private void writeMessageToDatastore(String message) throws IOException {
+    Entity messageEntity = new Entity("Message");
+
+    messageEntity.setProperty("message", message);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(messageEntity);
+  }
+
```

Comment:
Could you add a clarifying comment as to what this method is doing? I'm also a bit confused about the name. Can you explain in words what the function is doing and then try to be a little more specific with the name? Is it helping with get requests? If so, you could call it doGetHelper().

Reply:
For sure. That method is writing the comment the user inputs into a database (in this case datastore). Each instance of the saved comments in the database is called an Entity (that's why the name), but surely can think on something more specific